language/move-prover/interpreter/src/concrete/value.rs

```diff
@@ -1125,4 +1158,81 @@ impl EvalState {
             .or_insert_with(BTreeMap::new)
             .insert(partial_inst.ident, per_struct_map);
     }
+
+    pub fn load_memory(
```

Comment:
I would be nice to add to those public methods comments what they roughly do. In fact this coding style in model and prover, but it might not be uphold at all times. Still, we want to avoid adding more debt to this.

Similar applies BTW to the module headers. Here, the discipline is even less followed right now.

Reply:
Sure, I'll add the doc comments

Reply:
Comments added in the latest commit